django_bitmask_field.py

```diff
@@ -139,7 +139,7 @@ def get_prep_value(self, value):
             return value
         return int2bytes(value)
 
-    def from_db_value(self, value, expression, connection, context=None):
```

Comment:
why not to make this parameter like following?
```python
def from_db_value(self, value, expression, connection, *args, **kwargs):
```

Reply:
Fair enough, updated it.